src/test/kotlin/parts/wisdom/arcraider/ArcTest.kt

```diff
@@ -8,7 +8,8 @@ import java.io.File
 class ArcTest {
     @Test
     fun `load Task from file`() {
-        val file = File(arcData, "training/0a938d79.json")
+        val file = File(arcData, "extend_right.json")
```

Comment:
Switching to our own test file demonstrates that I shouldn't have used `arcData` here. I made some suggestions in another comment.